modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -62,6 +62,7 @@ class GrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   virtual std::size_t getNumberActiveGrains() const override;
   virtual Point getGrainCentroid(unsigned int grain_id) const override;
   virtual bool doesFeatureIntersectBoundary(unsigned int feature_id) const override;
+  virtual std::vector<unsigned int> getNewGrainIDs() const override;
```

Comment:
I'm not a big fan of filling in passed in vectors despite it usually being more efficient. I think this is cleaner until it becomes a problem with efficiency.